pandas/core/ops.py

```diff
@@ -1764,8 +1892,14 @@ def na_op(x, y):
             result = mask_cmp_op(x, y, op, (np.ndarray, ABCSeries))
         return result
 
-    @Appender('Wrapper for flexible comparison methods {name}'
-              .format(name=op_name))
+    if op_name in _op_descriptions:
+        op_desc = _op_descriptions[op_name]
+        doc = _flex_comp_doc_FRAME.format(desc=op_desc['desc'],
+                                          op_name=op_name)
+    else:
+        doc = "Flexible wrappers to comparison methods"
+
+    @Appender(doc)
```

Comment:
I was taking a look before merging, and there is no case when `op_name` is not in `_op_descriptions`. Any reason why this if was added?

Can we simply have?

```
@Appender(_flex_comp_doc_FRAME.format(op_name=op_name,
                                      desc=_op_descriptions[op_name]['desc'])
```

Or add an assert `op_name in _op_descriptions` if you think its useful, but I don't see this `if` adding value.